src/main/java/org/benetech/servicenet/service/UserService.java

```diff
@@ -318,6 +318,15 @@ public void removeNotActivatedUsers() {
         return authorityRepository.findAll().stream().map(Authority::getName).collect(Collectors.toList());
     }
 
+    /**
+     * @return true if any user is logged, and he has admin role
+     */
+    public boolean isAdmin() {
+        return getUserWithAuthorities().map(u -> u.getAuthorities().stream()
```

Comment:
isAdmin can be a method of the user or UserDTO entity

Reply:
ok